src/components/ceremony/handlers/delete-engagement-default-ceremony.handler.ts

```diff
@@ -0,0 +1,18 @@
+import { EventsHandler, IEventHandler } from '../../../core';
+import { EngagementDeletedEvent } from '../../engagement/events';
+import { CeremonyService } from '../ceremony.service';
+
+@EventsHandler(EngagementDeletedEvent)
+export class DetachEngagementRootDirectoryHandler
+  implements IEventHandler<EngagementDeletedEvent> {
+  constructor(private readonly ceremonies: CeremonyService) {}
+
+  async handle({ engagement, session }: EngagementDeletedEvent) {
+    // TODO Update DB is some fashion
+    const ceremonyId = engagement?.ceremony?.value;
+
+    if (ceremonyId) {
+      await this.ceremonies.delete(ceremonyId, session);
```

Comment:
Does the relationship to the engagement need to be set to false as well?

Reply:
I added relation delete.

Reply:
@michaelmarshall how is this being handled in the rest of the system? are we deleting relationships when they change or just setting them to false?

Reply:
@michaelmarshall says to just set the relationship `active` to false instead of deleting it.

Reply:
@CarsonF Ok, So did you fix it yourself?